src/main/java/com/redhat/ecosystemappeng/morpheus/service/FeedbackService.java

```diff
@@ -0,0 +1,44 @@
+package com.redhat.ecosystemappeng.morpheus.service;
+
+import com.redhat.ecosystemappeng.morpheus.model.FeedbackDto;
+import com.redhat.ecosystemappeng.morpheus.rest.FeedbackApi;
+import com.redhat.ecosystemappeng.morpheus.rest.ReportEndpoint;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.json.JsonObject;
+import jakarta.ws.rs.core.Response;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.jboss.logging.Logger;
+
+@ApplicationScoped
+public class FeedbackService {
+    private static final Logger LOGGER = Logger.getLogger(FeedbackService.class);
+    @Inject
+    @RestClient
+    FeedbackApi feedbackApi;
+
+    public void sendFeedback(FeedbackDto dto) {
+        LOGGER.debugf("FeedbackService DTO", dto.getComment());
+        feedbackApi.sendFeedback(dto);
+    }
+
+    public boolean checkFeedbackExists(String reportId) {
+        LOGGER.debugf("Checking if feedback exists for reportId: %s", reportId);
+        try {
+            Response response = feedbackApi.checkFeedbackExists(reportId);
+            if (response.getStatus() == 200) {
```

Comment:
It's Better to use constants
```suggestion
            if (response.getStatus() == Response.Status.OK.getStatusCode()) {
```